packages/dds/tree/src/test/shared-tree/fuzz/operationTypes.ts

```diff
@@ -11,110 +11,101 @@ export type Operation = TreeOperation | Synchronize;
 export type TreeOperation = TreeEdit | TransactionBoundary | UndoRedo | SchemaChange;
 
 export interface TreeEdit {
-	type: "edit";
-	contents: FieldEdit;
+	type: "treeEdit";
+	edit: FieldEdit;
 }
 
 export interface TransactionBoundary {
-	type: "transaction";
-	contents: FuzzTransactionType;
+	type: "transactionBoundary";
+	boundary: TransactionStart | TransactionAbort | TransactionCommit;
 }
 
 export interface UndoRedo {
 	type: "undoRedo";
-	contents: FuzzUndoRedoType;
+	operation: UndoOp | RedoOp;
 }
 
 export interface SchemaChange {
-	type: "schema";
-	contents: SchemaOp;
+	type: "schemaChange";
+	operation: SchemaOp;
 }
 
-export type FuzzFieldChange = FuzzInsert | FuzzRemove | FuzzMove;
-
 export interface FieldEdit {
 	type: "fieldEdit";
-	change: FieldEditTypes;
+	/** The field being edited */
+	field: FieldDownPath;
+	/** The edit performed on the field */
+	change: SequenceFieldEdit | RequiredFieldEdit | OptionalFieldEdit;
 }
 
-export interface FuzzInsert {
+export interface Insert {
 	type: "insert";
-	/**
-	 * DownPath to the field's parent node. Undefined iff this is the root trait.
-	 */
-	parent: DownPath | undefined;
-	/**
-	 * Key on the parent node corresponding to this field.
-	 */
-	key: FieldKey;
 	/**
 	 * Index to insert at within the field.
 	 */
 	index: number;
 	content: JsonableTree[];
 }
 
-export interface FuzzSet {
+export interface SetField {
 	type: "set";
 	/**
-	 * DownPath to the field's parent node. Undefined iff this is the root trait.
-	 */
-	parent: DownPath | undefined;
-	/**
-	 * Key on the parent node corresponding to this field.
-	 */
-	key: FieldKey;
-	/**
-	 * @privateRemarks - Optional fields use {@link FuzzRemove} to mean "remove the field's contents" rather than
+	 * @privateRemarks - Optional fields use {@link ClearField} to mean "remove the field's contents" rather than
 	 * a `FuzzSet` with undefined value, hence why this property is required.
 	 */
 	value: JsonableTree;
 }
 
-export type FieldEditTypes = SequenceFieldEdit | RequiredFieldEdit | OptionalFieldEdit;
-
 export interface SequenceFieldEdit {
 	type: "sequence";
-	edit: FuzzInsert | FuzzRemove | FuzzMove;
+	edit: Insert | FuzzRemove | IntraFieldMove;
```

Comment:
It looks like we are renaming all the operation types to remove the "Fuzz" prefix. "FuzzRemove" can maybe be renamed to something like "Remove" or "RemoveNode"

Reply:
Thanks! Looks like I missed that one.